docs/docs/concepts/Library_Configuration.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Library configuration"
+linkTitle: "Library configuration"
+weight: 6
+description: >
+    How to use configuration in JavaScript Functions
+---
+
+When writing a library you might want to be able to provide a loading configuration, so that different users can use the same library with slightly different behaviour (without changing the base code). For example, assuming you writing a library that adds `__last_updated__` field to a hash (you can see how it can also be done with [KeySpace triggers](./triggers/KeySpace_Triggers.md)), the code will look like this:
```

Comment:
```suggestion
When writing a library, you may want to provide a loading configuration so that different users can use the same library with slightly different behaviour, without changing the base code. For example, assume you write a library that adds a `__last_updated__` field to a hash (you can see how it can also be done with [KeySpace triggers](./triggers/KeySpace_Triggers.md)), the code will look like this:
```